### DISCUSSION TITLE
QLD Government

### DISCUSSION THREAD

**Discussion by batesbw** [eschweitzer78/gps-design-systems-lwc#376] [Q&A]
Hi Emmanuel,
Just wondering if there are any plans to extend this into QLD government?
Cheers
Ben

**Comment by eschweitzer78** [comment]
Hi Benjamin yes we have plans but haven’t started work yet as the latest release of the QLD DS is far superior to the previous ones but also very different. 
We expect to have something solid by the end of the calendar year, safe harbour.

**Reply by batesbw** [reply]
Thanks Emmanuel, I'll watch this space, much appreciated!

**Comment by eschweitzer78** [comment]
We have now support for QLD DS. Closing the discussion as a result.
